proxy/src/grpc/sql_query.rs

```diff
@@ -219,9 +234,17 @@ pub fn convert_output(
         Output::Records(batches) => {
             let mut writer = QueryResponseWriter::new(resp_compress_min_length);
             writer.write_batches(batches)?;
+            for batch in batches {
+                GRPC_HANDLER_COUNTER_VEC
+                    .query_succeeded_row
+                    .inc_by(batch.num_rows() as u64);
+            }
```

Comment:
It is hoped to be done outside `convert_output`. And collect total number of the rows and then `inc_by` it.